compiler/damlc/daml-visual/src/DA/Daml/Visual.hs

```diff
@@ -136,6 +136,8 @@ startFromUpdate seen world update = case update of
     LF.UGetTime -> Set.empty
     LF.UEmbedExpr _ upEx -> startFromExpr seen world upEx
     LF.UCreate tpl _ -> Set.singleton (ACreate tpl)
+    LF.UCreateInterface{} ->
```

Comment:
kind of tempted to try killing the visualizer in 2.0 but there is no chance I’ll convince people of that and as long as we don’t have to support new features it also does not get in the way all that much :shrug: